database/frontend/online-code-editors.json

```
@@ -152,6 +152,13 @@
     "url": "https://www.jdoodle.com/",
     "category": "frontend",
     "subcategory": "online-code-editors"
+  },
+  {
+    "name": "CodinGame",
+    "description": "A website to learn and practice programming with interactive challenges in different programming languages ",
```

[Proofreading] The description was a bit unclear. Consider using the below suggestion to help you: 
```suggestion
    "description": "This is a great website to learn and practice your coding skills. It provides interactive challenges in different programming languages ",
```